api/src/main/java/graphql/nadel/hooks/ServiceExecutionHooks.java

```diff
@@ -46,4 +48,16 @@ default NewVariableValue visitArgumentValueInQuery(HooksVisitArgumentValueEnviro
     ) {
         return CompletableFuture.completedFuture(Optional.empty());
     }
+
+    /***
+     * Called to resolve the service that should be used to fetch data for a field that uses dynamic service resolution.
```

Comment:
I'm also using "dynamic service resolution" in different places throughout the code. But I'm open to suggestions for a better term to describe this process of finding which service should be used for a field at runtime (or query time)

Reply:
name is good imho